plone/app/locales/locales/de/LC_MESSAGES/widgets.po

```diff
@@ -446,15 +446,15 @@ msgstr "Intern"
 
 #: ./patterns/resourceregistry/js/registry.js
 msgid "Internet Explorer conditional comment"
-msgstr ""
+msgstr "Bedingter Kommentar zum Internet Explorer"
```

Comment:
bad translation. This refers to a version-constraining comment for IE. like `<!--[if lt IE 7 ]>`